### PR TITLE
Removed Timber for logging and replaced with Log

### PR DESCRIPTION
**Pull request by mbcse**
Removed timber for logging and replaced with Log 
Solves #110

### CODE REVIEW

**Comment by thunderbiscuit** [comment]
Only two small comments:

1. Do you mind removing the dependency on Timber in the `app/build.gradle.kts` file? (line 85)
2. I see that you upgraded the Android Gradle plugin. It's a good idea but doesn't belong in this PR (also as a side note, this property is defined in the `buildSrc/src/main/kotlin/Dependencies.kt` file, and so that's where you'd go and fix it to 4.2.2). I assume this might have been done by your IDE as a recommendation and you just clicked "yes" to the upgrade. Not a problem at all but do you mind bringing it back to what it was before I merge? We'll upgrade it in a new commit.

File: build.gradle.kts [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:${Versions.androidGradlePlugin}")
+        classpath("com.android.tools.build:gradle:4.2.2")
```
Comment (thunderbiscuit) [inline build.gradle.kts:9]: This should be rolled back to the old line 
```gradle
classpath("com.android.tools.build:gradle:${Versions.androidGradlePlugin}")
```

See PR message for explanations (we'll upgrade the plugin in a different commit).

**Review by thunderbiscuit — CHANGES_REQUESTED** [review]
Good work! See small comment.

**Comment by mbcse** [comment]
> Good work! See small comment.

Done!

**Comment by thunderbiscuit** [comment]
Thanks a bunch!

**Pull request merged**